### PR TITLE
avm2: Stub flash.text.engine.GraphicElement

### PR DESCRIPTION
**Pull request by Aaron1011**

### CODE REVIEW

File: core/src/avm2/globals/flash/text/engine/GraphicElement.as [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package flash.text.engine {
+    import flash.display.DisplayObject;
+    import flash.events.EventDispatcher;
+    import __ruffle__.stub_constructor;
+
```
Comment (n0samu) [inline core/src/avm2/globals/flash/text/engine/GraphicElement.as:5]: Don't you need to import `__ruffle__.stub_method` ?

File: core/src/avm2/globals/flash/text/engine/GraphicElement.as [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            return 15.0;
+        }
+        public function set elementHeight(value:Number):void {
+            stub_setter("flash.text.engine.GraphicElement", "elementHeight");
+        }
+
+        public function get elementWidth():Number {
+            stub_getter("flash.text.engine.GraphicElement", "elementWidth");
+            return 15.0;
+
```
Comment (evilpie) [inline core/src/avm2/globals/flash/text/engine/GraphicElement.as:25]: _nit:_ empty line

File: core/src/avm2/globals/flash/text/engine/GraphicElement.as [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        }
+
+        public function set elementWidth(value:Number):void {
+            stub_setter("flash.text.engine.GraphicElement", "elementWidth");
+        }
+
+        public function get graphic():DisplayObject {
+            stub_getter("flash.text.engine.GraphicElement", "graphic");
+            return null;
+
```
Comment (evilpie) [inline core/src/avm2/globals/flash/text/engine/GraphicElement.as:35]: dito

**Review by n0samu — APPROVED** [review]

**Review by torokati44 — APPROVED** [review]
LGTM, but as with https://github.com/ruffle-rs/ruffle/pull/14694, I'm curious whether this fixed any issues or content, or is just for progress's sake (which is also okay of course).

**Comment by Aaron1011** [comment]
This is needed by the game 'Wladca Smokow'

**Pull request merged**